### PR TITLE
Refactored validators into separate files

### PR DESCRIPTION
**Pull request by Onager**
For ease of viewing and isolating

### CODE REVIEW

File: dftimewolf/lib/validators/manager.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    """Returns a list of all registered validators.
+
+    Returns:
+      A list of all registered validators.
+    """
+    return list(cls._validator_classes.keys())
+
+
+  @classmethod
+  def RegisterValidator(
```
Comment (tomchop) [inline dftimewolf/lib/validators/manager.py:24]: Other manages have a boolean parameter that allows for overwriting validators (useful if we have different internal validation logic)

**Review by tomchop — CHANGES_REQUESTED** [review]
a few questions, otherwise LGTM!

File: dftimewolf/lib/validators/gcp_zone.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+
+
+class GCPZoneValidator(AbstractValidator):
+  """Validates a GCP zone."""
+
+  # Source: https://cloud.google.com/compute/docs/regions-zones/
+  # Fetched 2023-01-13
+  # TODO - Fetch at runtime?
+  _zones = {
```
[the remark this answers is not in the document]
Comment (Onager) [reply dftimewolf/lib/validators/gcp_zone.py:17]: Ack, done.

File: dftimewolf/lib/validators/manager.py (hunk 1 above)
Comment (Onager) [reply dftimewolf/lib/validators/manager.py:24]: Added an override argument here

**Review by tomchop — APPROVED** [review]

**Pull request merged**